boltdesignsystem/bolt
# DS-771 Carousel breaks inside layout grid

## Jira

https://pegadigitalit.atlassian.net/browse/DS-771

## Summary

The carousel placed inside `.l-bolt-site` should stay centered without left or right misaligning.

## Details

The carousel should have an active slide centered horizontally in the viewport in all carousel variations such as these below:

- buttons inside
- buttons outside
- buttons inside with a property `overflow` set to `true`
- buttons outside with a property `overflow` set to `true`
- all slides per view examples (with buttons inside and outside, with and without `overflow` prop)

## How to test

Pull the beach. Go to the `/pattern-lab/?p=tests-carousel-in-layout` and see the carousel variations. Check if the carousel's active slide is always centered.
